storybook/README.md

```diff
@@ -0,0 +1,82 @@
+# Building Storybook with Webassembly and Qt 5.14
+
+
+
+## Configuring the environment
+### Install Emscripten v1.38.27
+	
+
+       # Get the emsdk repo
+	    git clone https://github.com/emscripten-core/emsdk.git
```

Comment:
is tthe indentation here ok? Isn't single tab sufficient for the entire block?

Reply:
Didn't notice it in the md viewer, but indeed looks like the entire block has an extra space compared to # Get the emsdk repo. Will update

Reply:
done